pkg/server/proto/server.proto

```diff
@@ -3698,6 +3698,10 @@ message StatusReport {
   // A health report of all of the resources that make up the report.
   // Copied from the plugin generated raw SDK status report for convenient API access.
   repeated Health deprecated_resources_health = 9;
+
+  // count of active entrypoint config connections. This is currently only
+  // applicable to deployment type status reports
+  uint32 entrypoint_config_connections = 14;
```

Comment:
I would just name this `instances_count` cause that's what we're measuring. If we change how we measure that in the future its less leaky than "entrypoint config connections."